DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -69,6 +69,14 @@ public enum SessionReplay {
                 description: "Datadog SDK must be initialized before calling `SessionReplay.enable(with:)`."
             )
         }
+
+        guard core.get(feature: SessionReplayFeature.self) == nil else {
```

Comment:
/change-request It is only checking the core given as parameter, but we should check all registered cores in [CoreRegistry](https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogInternal/Sources/CoreRegistry.swift).

Reply:
Absolutely! Thanks for the catch.